plugins/ruby-foreman-deface/debian/control

```diff
@@ -8,6 +8,6 @@ Homepage: https://github.com/DefaceCommunity/deface
 
 Package: ruby-foreman-deface
 Architecture: all
-Depends: ${misc:Depends}, bundler, foreman
+Depends: ${misc:Depends}, foreman (>= 3.1.0~)
```

Comment:
3.1 is the version introducing this helper and `foreman` already depends on `bundler` if needed.